pkg/generic/descriptor/http.go

```diff
@@ -67,6 +165,36 @@ func NewHTTPResponse() *HTTPResponse {
 	}
 }
 
+// SetStatusCode implements ResponseSetter.SetStatusCode of dynamicgo
+func (resp *HTTPResponse) SetStatusCode(code int) error {
+	resp.StatusCode = int32(code)
+	return nil
+}
+
+// SetHeader implements ResponseSetter.SetHeader of dynamicgo
+func (resp *HTTPResponse) SetHeader(key, val string) error {
+	resp.Header.Set(key, val)
+	return nil
+}
+
+// SetCookie implements ResponseSetter.SetCookie of dynamicgo
+func (resp *HTTPResponse) SetCookie(key, val string) error {
+	c := &http.Cookie{Name: key, Value: val}
```

Comment:
How do you set multi kv to cookie if you marshal it derectly?

Reply:
@jayantxie What is the scenario to have multi kv?
Is it like when there are multiple fields with api.cookie in an idl?

Reply:
according to [RFC](https://www.rfc-editor.org/rfc/rfc6265#section-4.1.1), multiple 'set-cookie' header is allowed

Reply:
@jayantxie You mean we need to create a string value according to the type of val (interface{}) in func (m *apiCookie) Response() before calling this SetCookie(), right?

Reply:
@jayantxie I'm thinking of keeping the Response func for api.cookie to return nil as before for now and setting the TODO to do SetCookie.